contracts/staking/StakeManager.sol

```diff
@@ -102,16 +118,26 @@ contract StakeManager is Validator, IStakeManager, RootChainable, Lockable {
     NFTCounter = NFTCounter.add(1);
```

Comment:
For `emit Staked` line just above it, I think using `currentEpoch` might be more gas efficient that using `validators[NFTCounter].activationEpoch`. Not sure. Please check.

Reply:
both are same actually but `currentEpoch` might look clean. what say?